### ISSUE TITLE
Accessibility: Implement the proper Button Menu design pattern for the main menu

### ISSUE THREAD

**Issue by MarcoZehe** [deltachat/deltachat-desktop#2139]
- Operating System (Linux/Mac/Windows/iOS/Android): All desktop
- Delta Chat Version: 1.14
- Expected behavior: The main menu would work with screen readers and keyboard.
- Actual behavior: The "menu" opens, but has no menu semantics, no focus management and no keyboard support.
- Steps to reproduce the problem: Open DeltaChat, tab to the Main menu button and press Space or Enter to open. Expected behavior is that focus moves to the menu and you can use up and down arrows to choose, enter to select/execute the item, much like the context menus work.
- Screenshots: N/A
- Logs: N/A

## Proposed solution

Implement the [menu button](https://www.w3.org/TR/wai-aria-practices-1.1/#menubutton) design pattern from the WAI-ARIA authoring practices.

**Comment by WofWca** [comment]
Apologies, this is actually about the three-dot menu. And I might have misremembered. Did we even have that sidebar??

Here is what it looks like now:

![image](https://github.com/user-attachments/assets/f835f443-063c-48e2-b128-ef99aa48fde4)

And here is what it used to look like in 1.14:

![image](https://github.com/user-attachments/assets/3618d4f4-7ba5-4ae0-8b99-7bb7e8668bdc)

The current version actually has better accessibility and utilizes the same context menu we use for other context menus. Keyboard-accessibility is present, and `role="menuitem"` is used.
So I suppose let's close it.

**Comment by Simon-Laux** [comment]
There was a sidebar for some versions. it was removed around when I added the account switcher sidebar.
